README.md

```diff
@@ -98,7 +98,7 @@ outputs = { self, flake-utils, nixpkgs-terraform, nixpkgs }:
     in
     {
       devShells.default = pkgs.mkShell {
-        buildInputs = [ pkgs.terraform-versions."1.6.3" ];
+        buildInputs = [ pkgs.terraform-versions."1.7.3" ];
```

Comment:
@oscar-izval I am wondering if we should leave a placeholder (`x.y.z`) in this examples, followed by a link to the `versions.json` file to reduce some maintenance efforts. What do you think?